src/components/imageUploader/imageUploader.js

```diff
@@ -1,25 +1,41 @@
-define(['dialogHelper', 'connectionManager', 'dom', 'loading', 'scrollHelper', 'layoutManager', 'globalize', 'require', 'emby-button', 'emby-select', 'formDialogStyle', 'css!./style'], function (dialogHelper, connectionManager, dom, loading, scrollHelper, layoutManager, globalize, require) {
-    'use strict';
-
-    var currentItemId;
-    var currentServerId;
-    var currentFile;
-    var hasChanges = false;
-
-    function onFileReaderError(evt) {
+/* eslint-disable indent */
+
+/**
+ * Module for imageUploader.
+ * @module components/imageUploader/imageUploader
+ */
```

Comment:
These comments at the beginning of the files seem pretty useless in general since they don't actually include any extra information about the files themselves.

Reply:
The sentence should generally be a more useful description of what the file does.

Like:
```javascript
/*
 * Handles the metadata editing dialog and its related actions
 * @module whatever/the/real/path/is
 */
```